stubs/inertia-vue-ts/resources/js/Components/Modal.vue

```diff
@@ -0,0 +1,96 @@
+<script setup lang="ts">
+import { computed, onMounted, onUnmounted, watch } from 'vue';
+
+const props = withDefaults(
+    defineProps<{
+        show?: boolean;
+        maxWidth?: '2xl';
```

Comment:
shouldn't this be `'sm' | 'md' | 'lg' | 'xl' | '2xl'`?

Reply:
Yes, thank you!